gatsby/src/layouts/default-layout.tsx

```diff
@@ -22,10 +22,7 @@ const DefaultLayout: React.FC = ({ children }) => {
         ]}
       />
       <main className={styles.main}>
-        <MainBackground
-          src="/images/main-content-mobile-bg.jpg"
-          alt="Covid Portal"
-        />
+        <MainBackground src="/images/mainbg.png" alt="Covid Portal" />
```

Comment:
Alt text by měl popisovat co na té fotce je… A v tomhle případě, když je spíš dekorativní, je lepší když je tam jen atl="". Tím ho čtečky přeskočí, ale splníš to že tag je required.